public/index.html

```diff
@@ -2,7 +2,6 @@
 <html lang="en">
```

Comment:
![image](https://github.com/lorenzoworx/math/assets/93039359/1c8ab6be-804e-4b01-b3be-3c6f7880d749)

- [ ] It would be nice if you fix the Deployment link in the Table of Contents because right now this link is broken :fearful: Kindly fix it to make your readme descriptive and professional :+1: